command/operator_root.go

```diff
@@ -0,0 +1,31 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type OperatorRootCommand struct {
+	Meta
+}
+
+func (*OperatorRootCommand) Help() string {
+	helpText := `
+	Usage: nomad operator root [options]
+
+	This command is accessed by using one of the subcommands below.
+	`
+
+	return strings.TrimSpace(helpText)
+}
+
+func (*OperatorRootCommand) Synopsis() string {
+	return "Provides access to root encryption keys"
+}
+
+func (f *OperatorRootCommand) Name() string { return "operator gossip" }
+
+func (f *OperatorRootCommand) Run(args []string) int {
```

Comment:
The args function parameter is unused, so we can use an underscore to show this is ignored.

```suggestion
func (f *OperatorRootCommand) Run(_ []string) int {
```